### PR TITLE
Editing

### PR DESCRIPTION
**Pull request by danicadombek**
- Add edit-function to DetailMemoryPages and App.js (without tests and storybook yet)
- Warning: Updated styling - max. a small look necessary :)

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
-import PropTypes from 'prop-types'
+// import PropTypes from 'prop-types'
```
Comment (roscha80) [inline src/components/Header.js]: You can delete this. You dont got any props anymore.

File: src/components/Header.js [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
 import styled from 'styled-components/macro'
+import logo from '../assets/logo.png'
 
-Header.propTypes = {
-  children: PropTypes.node.isRequired,
-}
+// Header.propTypes = {
+//   children: PropTypes.node.isRequired,
+// }
```
Comment (roscha80) [inline src/components/Header.js]: You can delete this. You dont got any props anymore. :)

File: src/components/IconButton.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types'
+import Button from './Button'
+// import DeleteIcon from '../assets/images/icons/delete.png'
```
Comment (roscha80) [inline src/components/IconButton.js:3]: You can delete this :)

File: src/components/IconButton.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  onClick: PropTypes.func,
+}
+
+export default function IconButton(props) {
+  return <IconButtonStyle {...props} />
+}
+const IconButtonStyle = styled(Button)`
+  background: none;
+  box-shadow: none;
+`
```
Comment (roscha80) [inline src/components/IconButton.js:16]: Maybe u call it DeleteIconButton ;) Its more specific.

File: src/pages/MemoriesPage.js [hunk 5]
```diff
@@ -1,14 +1,16 @@
 //@ts-check
 import Button from '../components/Button'
-import DeleteButton from '../components/DeleteButton'
+import IconButton from '../components/IconButton'
```
Comment (roscha80) [inline src/pages/MemoriesPage.js:3]: Again, i think DeleteIconButton or like it was DeleteButton is more specific :)

**Review by roscha80 — APPROVED** [review]
Good work Danica. Is looking nice. I only got small mentions :)

File: src/components/IconButton.js (hunk 4 above)
Comment (danicadombek) [reply src/components/IconButton.js:16]: I have two IconButtons: Delete and edit :)

**Pull request merged**